test/controllers/authentications_controller_test.rb

```diff
@@ -161,33 +197,51 @@
 
     describe 'redirect_matching_account' do
       it 'must sign in an existing user through github' do
-        github_stub = stub(email: account.email, login: account.login, access_token: Faker::Lorem.word)
-        @controller.stubs(:github_api).returns(github_stub)
+        @controller.stubs(:github_api).returns(github_account_stub)
 
         get :github_callback, code: Faker::Lorem.word
 
         account.reload
         must_redirect_to account
         request.env[:clearance].current_user.id.must_equal account.id
-        account.github_verification.token.must_equal github_stub.access_token
+        account.github_verification.token.must_equal github_account_stub.access_token
+      end
+
+      it 'must sign in a verified user regardless of github restrictions' do
+        github_account_stub.stubs(:created_at).returns(Time.current)
+        @controller.stubs(:github_api).returns(github_account_stub)
```

Comment:
This test explains that once a user is github verified by us, we would sign them in regardless of their github activity. We do this to ensure a snappy sign in.